modules/graceful-shutdown.adoc

```diff
@@ -17,6 +17,7 @@ You can shut down a cluster until a year from the installation date and expect i
 
 * You have access to the cluster as a user with the `cluster-admin` role.
 * You have taken an etcd backup.
+* If you are running a {sno} cluster, you must evacuate all workload pods before you shut it down.
```

Comment:
@vrutkovs updated here to use the same wording as the other PR. Let me know if it sounds okay to you